src/views/TabView/TabView.js

```diff
@@ -127,6 +127,8 @@ class TabView extends PureComponent<void, Props, void> {
     );
   };
 
+  _renderPager = (props: *) => <TabViewPagerPan {...props} />;
```

Comment:
This looks like it's rendering the pan shadow (from TabViewPagerPan) component on the normal iOS TabView even when swiping is disabled. Should this use the same platform-specific logic from https://github.com/react-community/react-navigation/commit/0cd3eaa97afe9806f7abe31785ad13f75d786542#diff-6f52f52094e1001dd754981e92d9f933L55 to determine which component to render?

That, or should the condition below on L165 be `animationEnabled === false && swipeEnabled === true` so we only show the pager when the view can be swiped?

Patched both of these solutions locally and both seem to fix the issue; not sure which one is correct.